client.ts

```diff
@@ -17,13 +17,28 @@ export class Client {
 
   // TODO: can we use more specific type for args?
   async query(
-    text: string | QueryConfig,
-    ...args: any[]
+    text: string,
+    args?: unknown[],
   ): Promise<QueryResult> {
-    const query = new Query(text, ...args);
+    const query = new Query({ text, args });
     return await this._connection.query(query);
   }
 
+  async multiQuery(
+    text: string | string[],
+    args?: unknown[],
+  ): Promise<QueryResult[]> {
+    if (!Array.isArray(text)) text = text.split(";").map((el) => el.trim());
```

Comment:
This line seems brittle; can you add some tests showcasing that it works as intended?

Reply:
hehe, yeah, currently there are a lot of tests failing, will fix all the tests when we have decided on the final implementation 💪